### ISSUE TITLE
wrong parts numbers in extended BOM

### ISSUE THREAD

**Issue by lifthoofd** [WorldWilderLab/PlanEt_SMD#6]
K1 and K2 have the 2 lead connectors as parts numbers, correct part number should be 640457-3

**Comment by ghost** [comment]
Correct, it should indeed be the -3 part.

http://uk.farnell.com/te-connectivity-amp/640457-3/header-right-angle-0-1-3way/dp/588726?ost=640457-3&categoryId=&categoryName=

**Comment by lifthoofd** [comment]
can you adjust the partlist in the repo so its correct on this point?

**Comment by ghost** [comment]
yes, I'll try to find out how I got it in there in the first place !
